talib/abstract.py

```diff
@@ -0,0 +1,34 @@
+from ._ta_lib import Function as _Function, __TA_FUNCTION_NAMES__
+
+__ABSTRACT_FUNCTION_NAMES__ = ["_get_defaults_and_docs"]
+
+_ta_func = __import__(
```

Comment:
Why do you use ``__import__`` instead of just ``import _ta_lib`` and use it directly?